scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,430 @@
+    'questions-list.constants.ts',
+    'read-only-exploration-backend-api.service.ts',
+    'replace-inputs-with-ellipses.filter.ts',
+    'require-is-float.directive.ts',
+    'review-test-backend-api.service.ts',
+    'review-test-domain.constants.ajs.ts',
+    'review-test-domain.constants.ts',
+    'review-test-page.constants.ajs.ts',
+    'review-test-page.constants.ts',
+    'richTextComponentsRequires.ts',
+    'rich_text_components_definitions.ts',
+    'search-explorations-backend-api.service.ts',
+    'services.constants.ajs.ts',
+    'services.constants.ts',
+    'skill-domain.constants.ajs.ts',
+    'skill-domain.constants.ts',
+    'skill-editor-page.constants.ajs.ts',
+    'skill-editor-page.constants.ts',
+    'skill-mastery-backend-api.service.ts',
+    'skills-mastery-list.constants.ajs.ts',
+    'skills-mastery-list.constants.ts',
+    'state-content.service.ts',
+    'state-editor.constants.ajs.ts',
+    'state-editor.constants.ts',
+    'state-interaction-id.service.ts',
+    'state-recorded-voiceovers.service.ts',
+    'state-rules-stats.service.ts',
+    'state-solicit-answer-details.service.ts',
+    'state-solution.service.ts',
+    'state-top-answers-stats-backend-api.service.ts',
+    'state-written-translations.service.ts',
+    'statistics-domain.constants.ajs.ts',
+    'statistics-domain.constants.ts',
+    'story-domain.constants.ajs.ts',
+    'story-domain.constants.ts',
+    'story-editor-page.constants.ajs.ts',
+    'story-editor-page.constants.ts',
+    'story-viewer-domain.constants.ajs.ts',
+    'story-viewer-domain.constants.ts',
+    'subtopic-viewer-backend-api.service.ts',
+    'subtopic-viewer-domain.constants.ajs.ts',
+    'subtopic-viewer-domain.constants.ts',
+    'summarize-nonnegative-number.filter.ts',
+    'topic-domain.constants.ajs.ts',
+    'topic-domain.constants.ts',
+    'topic-editor-page.constants.ajs.ts',
+    'topic-editor-page.constants.ts',
+    'topic-landing-page.constants.ajs.ts',
+    'topic-landing-page.constants.ts',
+    'topic-viewer-domain.constants.ajs.ts',
+    'topic-viewer-domain.constants.ts',
+    'topics-and-skills-dashboard-domain.constants.ajs.ts',
+    'topics-and-skills-dashboard-domain.constants.ts',
+    'topics-and-skills-dashboard-page.constants.ajs.ts',
+    'topics-and-skills-dashboard-page.constants.ts',
+    'translation-status.service.ts',
+    'translation-tab-active-content-id.service.ts',
+    'translation-tab-active-mode.service.ts',
+    'truncate-at-first-ellipsis.filter.ts',
+    'truncate-at-first-line.filter.ts',
+    'uiLeafletRequires.ts',
+    'underscores-to-camel-case.filter.ts',
+    'undo-redo.service.ts',
+    'user-exploration-permissions.service.ts',
+    'valueGeneratorsRequires.ts',
+    'wrap-text-with-ellipsis.filter.ts',
+]
+
+
+class LcovStanzaRelevantLines:
+    """Provides relevant data of a lcov stanza in order to calc any frontend
+    test coverage decreasement.
+    """
+
+    def __init__(self):
```

Comment:
Take the stanza lines as an arg here and initialize the member variables directly; don't call the set() functions separately.

I.e. the logic for set_file_name, set_total_lines, etc. should all be moved here, since without that info an instance of this class doesn't really have any meaning.

Reply:
Done